x/escrow/init.go

```diff
@@ -0,0 +1,62 @@
+package escrow
+
+import (
+	"encoding/hex"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/x/cash"
+	"github.com/pkg/errors"
+)
+
+var _ weave.Initializer = (*Initializer)(nil)
+var burnAddress, _ = hex.DecodeString("0000000000000000000000000000000000000000")
```

Comment:
What about `make([]byte, weave.AddressLength)`, which would be programatically checkable and all 0 initialized anyway